app/main/views/audits.py

```diff
@@ -39,3 +67,32 @@ def list_audits():
             request.args
         )
     )
+
+
+@main.route('/audit-events/<string:audit_id>/acknowledge', methods=['POST'])
```

Comment:
I don't really mind either status. We do use `<int:...>` for suppliers and users though, and it looks slightly easier than the `try: int(...) ...` check.